translations/translations.json

```
@@ -5311,5 +5311,15 @@
     "zh-CN": {
       "pageTitle": "版本说明和联系方式- {{appName}}"
     }
+  },
+  "pois": {
+    "de": {
+      "location": "Standort",
+      "map": "Karte"
+    },
+    "en": {
+      "location": "Location",
+      "map": "Map"
+    }
```

Only German is enough :) English should be translated by professionals as well
```suggestion
    }
```